src/pages/tutorials/backend/create-custom-rest-api.md

```diff
@@ -98,7 +98,20 @@ Where:
 *  `service` describes the interface and the method to be called when the endpoint is reached.
 *  `resource` sets the ACL resource that is required to access the endpoint. If you want to make it public (no authentication), use: `<resource ref="anonymous" />`
 
-## Step 4. Create interfaces
+## Step 4. Create di.xml
+
+Create di.xml in `app/code/Dev/RestApi/etc/di.xml` to configure interfaces
+
+```xml
+<?xml version="1.0" ?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <preference for="Dev\RestApi\Api\ProductRepositoryInterface" type="Dev\RestApi\Model\Api\ProductRepository"/>
+    <preference for="Dev\RestApi\Api\RequestItemInterface" type="Dev\RestApi\Model\Api\RequestItem"/>
+    <preference for="Dev\RestApi\Api\ResponseItemInterface" type="Dev\RestApi\Model\Api\ResponseItem"/>
+</config>
+```
+
```

Comment:
Hi @chakravarthykalyan90,

I would recommend mentioning either here, or before the step `Test your custom endpoint` that after manipulating .xml files mentioned in this topic, and/ or updating constructors or introducing new classes, it is required to execute `bin/magento setup:upgrade` (https://experienceleague.adobe.com/docs/commerce-operations/installation-guide/tutorials/database-upgrade.html) or `bin/magento setup:di:compile` (https://experienceleague.adobe.com/docs/commerce-operations/configuration-guide/cli/code-compiler.html) from the directory where Magento Open Source is installed in order to generate the code for the introduced changes.

Reply:
Thank you, @chakravarthykalyan90